### DISCUSSION TITLE
Rabbit MQ : inet_error,{tls_alert,{user_canceled,"TLS server: In state connection received CLIENT ALERT: Fatal - User Canceled

### DISCUSSION THREAD

**Discussion by christian-dev** [rabbitmq/rabbitmq-server#10060] [Questions]
Hello, 
I have frequently this message in the rabbitmq logs : 
**{inet_error,{tls_alert,{user_canceled,"TLS server: In state connection received CLIENT ALERT: Fatal - User Canceled\n"}}}**
_rabbitMQ Version_  : 3.12.8

Could someone knows the possible root cause ?
Thanks and regards

**Comment by lukebakken** [comment]
* What version of Erlang are you using?
* Which operating system and version are you using?

**Reply by christian-dev** [reply]
Hello, 
My erlang version is  22.0.7
and the operating system is Linux 4.18.0-425.3.1.el8.x86_64 x86_64
Thanks you for your helps

**Reply by michaelklishin** [reply]
@christian-dev RabbitMQ 3.12 [cannot run on Erlang 22](https://rabbitmq.com/which-erlang.html). The last version to support Erlang 22 was `3.8.15`, which has [long been out of support](https://rabbitmq.com/which-erlang.html).

**Reply by michaelklishin** [reply]
`user_cancelled` means the client side of a TLS connection [has terminated the handshake process](https://techcommunity.microsoft.com/t5/iis-support-blog/ssl-tls-alert-protocol-and-the-alert-codes/ba-p/377132).

We cannot know why that happened but we do have a [methodology that helps narrow down TLS connectivity issues](https://rabbitmq.com/troubleshooting-ssl.html).
